Utilities/CodeGeneration/EdFi.Ods.CodeGen/Options.cs

```diff
@@ -21,5 +22,8 @@ public class Options
 
         [Option('i', "includePlugins", Required = false, HelpText = "Flag to determine if plugin assemblies are included", Default = false)]
         public bool IncludePlugins { get; set; }
+
+        [Option('j', "extensionLocationPaths", Required = false, HelpText = "Array of paths for the Extension location to determine if plugin assemblies are included")]
```

Comment:
is 'j' the right parameter? what other options do we have?

Reply:
should the long parameter be `extensionsPaths`

Reply:
> is 'j' the right parameter? what other options do we have?

I changed to extensionsPaths . After that  already we have e parameter for engine . Can I use  'p' ?